src/Groups/GAPGroups.jl

```diff
@@ -1065,7 +1066,7 @@ Return `C, f`, where `C` is the normal core of `H` in `G`,
 that is, the largest normal subgroup of `G` that is contained in `H`,
 and `f` is the embedding morphism of `C` into `G`.
 """
-core(G::T, H::T) where T<:GAPGroup = _as_subgroup(G, GAPWrap.Core(G.X, H.X))
+core(G::T1, H::T2) where {T1<:GAPGroup, T2<:GAPGroup} = _as_subgroup(G, GAPWrap.Core(G.X, H.X))
```

Comment:
```suggestion
core(G::GAPGroup, H::GAPGroup) = _as_subgroup(G, GAPWrap.Core(G.X, H.X))
```
Your change makes it actually a lot easier

Reply:
It makes the signatures easier (with Lars' extra change). Of course we loose a bit of "validation", i.e. previously Julia prevented some situations from occurring (e.g. you could not mix a permutation group with a matrix group), and now we have to catch that ourselves. But that's not so bad in the end, we already have to catch some of these "bad" situations, e.g. when both groups are matrix groups but defined over different rings (perhaps in different characteristic)